make/maven-docker-build/build.sh

```diff
@@ -62,13 +62,13 @@ for k in "${specialproject[@]}"
 do
 	cd $SOURCEDIR/$k
 	mvn clean initialize  >> $BUILDLOG
-	mvn package  >> $BUILDLOG
+	mvn package -Dmaven.test.skip  >> $BUILDLOG
```

Comment:
why remove the test part?

Reply:
For the build:

Reply:
Reason 1: The build take a lot time stucking by the UT. 
Reason 2: For flowgate-api, The UT use a Couchbase which would sleep 200s waiting for initialize,  and have a certain probability to fail.  This also could lead to out of time Error in build. 
Reason 3: The UT have been run at the PR. 
So I think it's better to skip the test.

Reply:
Reverted. 
The UT in flowgate-api use a Couchbase docker image which would sleep 200s waiting for initialize,  and have a certain probability to fail.  I want to skip test to avoid this at first.  
But now, I figure out the skip would fail the generation of api-guide doc resources.